Makefile

```diff
@@ -133,10 +135,25 @@ endif
 
 .PHONY: integration-test
 # Run integration tests
-integration-test:  ## Runs integration tests
-	./hack/run-integration-tests.sh
+integration-test: envtest  ## Runs integration tests
+	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" ./hack/run-integration-tests.sh
 
 # Run e2e tests
 .PHONY: e2e-test
-e2e-test: cli
-	./hack/run-e2e-tests.sh
+e2e-test: cli envtest
```

Comment:
the `e2e-test` is not a rerun of the integration tests... I think there was some confusion around this.   The e2e tests should NOT be dependent on the `envtest`.  e2e requires docker + kind.    Although the refactor of the circleci -> git action lost that.... which is why we continue to have a circleci task along with the github actions.   I'm not sure we need to solve all of that in this PR but we should revert this portion.   I will wait a day for a response... if none, I will merge and mod, then create a PR for the e2e to be on github actions (assuming it's possible... It will be an area I will need to research).

Reply:
I've removed the `e2e-test` make target.